softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRouterHandler.java

```diff
@@ -4,7 +4,7 @@
 import com.softgroup.common.protocol.Response;
 import org.springframework.beans.factory.annotation.Autowired;
 
-public abstract class AbstractRouterHandler<T extends Handler>
+public abstract class AbstractRouterHandler<T extends RequestHandler>
```

Comment:
@AlexKazmirchuk why RequestHandler? Router can work with request handlers and with another routers.